_includes/registration-closed-overlay.html

```diff
@@ -29,6 +31,13 @@
             </div>
         </div>
     </div>
+    <script>
```

Comment:
Das ist etwas redundant, oder? Ich meine, warum packst Du das nicht in das bestehende Skript zur deadline mit rein?

Reply:
Hab es spontan nicht hinbekommen und dann scheinbar vergessen nachzuholen.